Tests/AsyncHTTPClientTests/HTTPClientTestUtils.swift

```diff
@@ -59,6 +64,19 @@ let canBindIPv6Loopback: Bool = {
     return didBind
 }()
 
+/// Runs the given block in the context of a non-English C locale (in this case, German).
+/// Throws an XCTSkip error if the locale cannot be set.
+func withCLocaleSetToGerman(_ body: () throws -> Void) throws {
+    guard let germanLocale = newlocale(LC_TIME_MASK | LC_NUMERIC_MASK, "de_DE", nil) else {
+        throw XCTSkip("Unable to create locale de_DE")
```

Comment:
This is aggressive: I think we only want to skip the test if errno was `ENOENT`. Otherwise the error is in our code and we should know about it by `XCTFail`.